### DISCUSSION TITLE
Add system tray icon

### DISCUSSION THREAD

**Discussion by WilliUreta** [LizardByte/community#72] [Sunshine] [answered]
Is there a way to hide the Sunshine window in the tray, similar to what RTX Experience has while running. Maybe the close button in windows "minimizes" it to the tray. 
I feel it would make a "cleaner" look instead of having a cmd-like info always there. 
I've seen this mentioned among other stuff here: https://github.com/LizardByte/Sunshine/discussions/224 ,but I don't quite understand why the systray should be implemented in all OSes

Thanks

**Comment by ReenigneArcher** [comment]
> I don't quite understand why the systray should be implemented in all OSes

Windows is not the only OS with a "system tray". macOS and Linux (depending on the DE) have them as well, so if a system tray icon is added it should be cross platform.

I would accept a PR that adds a system tray icon, although if you run Sunshine as a service you won't see the system tray icon anyway.

Edit: I found this cross platform system tray library if anyone wants to make an attempt at implementing it. https://github.com/dmikushin/tray

**Reply by Movieaholic** [reply]
> > I don't quite understand why the systray should be implemented in all OSes
> 
> Windows is not the only OS with a "system tray". macOS and Linux (depending on the DE) have them as well, so if a system tray icon is added it should be cross platform.
> 
> I would accept a PR that adds a system tray icon, although if you run Sunshine as a service you won't see the system tray icon anyway.

How is this "Run Sunshine as a service" achieved? i ran the program in the tools file and nothing happened?

**Reply by ReenigneArcher** [reply]
> How is this "Run Sunshine as a service" achieved? i ran the program in the tools file and nothing happened?

There are bat files to enable or disable the service. These are missing in the current release, but has been fixed for the next release.

**Comment by ReenigneArcher** [comment, marked as answer]
Tray icon is part of Sunshine for last several releases. Additionally, tray notifications are coming in the next release.

Regarding the rest of your comment, be sure to run Sunshine as a service (the default for the Windows installer).
